content/the-ember-times-issue-189.md

```diff
@@ -66,15 +68,13 @@ If this is something you are interested in, please read and discuss the [Asset I
 
 ---
 
-## [🐹 4. Section title in sentence case](section-url)
+## [🏛️ EmberFest 2021 - best of both worlds](https://emberfest.eu)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+Next week after a year of absence because of COVID [EmberFest](https://emberfest.eu) is back. On the 30th of September and the 1st of October Emberistas will gather **digitally** and **analogue** in Rome to hear awesome Ember-talks.
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+With well-known EmberConf/Fest speakers like [Melanie Sumner (@MelSumner)](https://github.com/MelSumner) and [Edward Faulkner (@ef4)](https://github.com/ef4) and first time speakers like [Bartlomiej Dudzik (@BobrImperator)](https://github.com/BobrImperator), [Jenny Judova (@JennyJudova)](https://github.com/JennyJudova) and [Zoë Bijl (@ZoeBijl)](https://github.com/ZoeBijl) the [line up](https://emberfest.eu/schedule/) is super diverse and interesting for all Ember enthousiasts.
```

Comment:
I think it'll help to break a long sentence into two. How about:

```suggestion
There will be well-known EmberConf/Fest speakers like [Melanie Sumner (@MelSumner)](https://github.com/MelSumner) and [Edward Faulkner (@ef4)](https://github.com/ef4), as well as first time speakers like [Bartlomiej Dudzik (@BobrImperator)](https://github.com/BobrImperator), [Jenny Judova (@JennyJudova)](https://github.com/JennyJudova), and [Zoë Bijl (@ZoeBijl)](https://github.com/ZoeBijl). The [line-up](https://emberfest.eu/schedule/) is super diverse and interesting for all Ember enthusiasts.
```

Reply:
Great suggestion :)